lib/kino/audio.ex

```diff
@@ -0,0 +1,74 @@
+defmodule Kino.Audio do
+  @moduledoc """
+  A kino for rendering a binary audio.
+
+  ## Examples
+
+      content = File.read!("/path/to/audio.wav")
+      Kino.Audio.new(content, :wav)
+
+      content = File.read!("/path/to/audio.wav")
+      Kino.Audio.new(content, :wav, [:controls, :autoplay, :loop])
+  """
+
+  use Kino.JS
+  use Kino.JS.Live
+
+  @type t :: Kino.JS.Live.t()
+
+  @type mime_type :: binary()
+  @type common_audio_type :: :wav | :mp3 | :mpeg | :ogg
+  @doc """
+  Creates a new kino displaying the given binary audio.
+
+  The given type be either `:wav`, `:mp3`/`:mpeg`, `:ogg`
+  or a string with audio MIME type.
+
+  """
+  @spec new(binary(), common_audio_type() | mime_type(), list()) :: t()
+  def new(content, type, opts \\ [:controls, :autoplay]) when is_binary(content) do
+    Kino.JS.Live.new(__MODULE__, %{
+      content: content,
+      type: mime_type!(type),
+      opts: Enum.join(opts, " ")
+    })
+  end
```

Comment:
Actually there are 5 options you can pass here: `autoplay`, `controls`,  `loop `, `muted` ,`preload`. So list a bit shorter.

Reply:
I don't think `preload` matters, let's support `autoplay`, `loop` and `muted` for now. I still prefer regular options, because this gives us more control, and we may have other options in the future. Also with the current implementation the user could technically interpolate anything :)